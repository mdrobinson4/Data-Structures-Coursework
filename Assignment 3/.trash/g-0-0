13 11 24 ;
11 12 40 ;
15 7 42 ;
13 6 26 ;
9 12 43 ;
12 8 56 ;
7 14 61 ;
15 8 18 ;
8 9 92 ;
11 15 19 ;
8 14 96 ;
13 9 53 ;
7 8 52 ;
9 7 57 ;
10 13 60 ;
10 11 40 ;
9 12 47 ;
6 12 73 ;
8 6 50 ;
7 11 82 ;
