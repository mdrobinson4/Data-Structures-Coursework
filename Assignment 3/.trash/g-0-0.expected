0: 6
1: 13(6)26
2: 11(13)24
3: 15(11)19 10(11)40 12(11)40
4: 8(15)18 7(15)42 9(12)47
5: 14(7)61
weight: 317
